package.json

```diff
@@ -35,6 +35,7 @@
     "clean": "npm cache clean && npm run rimraf -- node_modules doc typings coverage dist .awcache && npm run clean:compile",
     "clean:dist": "npm run rimraf -- dist .awcache",
     "clean:compile": "npm run rimraf -- \"src/compiled\" ",
+    "clean:overlaycss": "npm run rimraf -- \"node_modules/@angular/material/core/overlay/overlay.css\" ",
```

Comment:
By default, node-sass will throw an error about not being able to tell if it should import overlay.css or overlay.scss when you try to compile the material themes unless you delete overlay.css or manually edit the _core.scss file.